irc.html

```diff
@@ -13,21 +13,21 @@ <h1>Channels</h1>
   </p>
   <ul>
     <li>
-      <strong><a href="https://webchat.freenode.net/?channels=#minetest">#minetest</a></strong>
-      (<a href="http://irc.minetest.net/minetest">logs</a>): main channel
+      <strong><a href="https://webchat.freenode.net/?channels=#minetest">#Minetest</a></strong>
+        (<a href="http://irc.minetest.net/minetest">Logs</a>):
+        Main channel, where players can join to chat and get support about Minetest.
     </li>
+    
     <li>
-      <strong>
-        <a href="https://webchat.freenode.net/?channels=#minetest-dev">#minetest-dev</a>
-      </strong>
-      (<a href="http://irc.minetest.net/minetest-dev">logs</a>):
-      development channel, only serious stuff; note that webchat is muted due to too much spam/flood.
+      <strong><a href="https://webchat.freenode.net/?channels=#minetest-dev">#Mminetest-Dev</a></strong>
+        (<a href="http://irc.minetest.net/minetest-dev">Logs</a>):
+        Development channel, for topics related to bug reporting and general Minetest development.
     </li>
+    
     <li>
-      <strong>#minetest-hub</strong>
-      (<a href="http://irc.minetest.net/minetest-hub">logs</a>):
-      A moderated channel for people who regularly improve Minetest by developing, modding, hosting servers and otherwise.
-      See <a href="http://hub.minetest.net/info">the channel's website</a> for more information.
+      <strong><a href="https://webchat.freenode.net/?channels=#minetest-hub">#Minetest-Hub</a></strong>
+        (<a href="http://irc.minetest.net/minetest-hub">Logs</a>):
+        Community channel, to talk about modding, server adminitration and short off-topic.
```

Comment:
The channel is not for offtopic; it's simply something we usually accept. I prefer the previous description for -hub, then adjust the other two so they are full sentences.

Reply:
And what about the description for the other channels?